hack/update-codegen-clients.sh

```diff
@@ -19,29 +19,20 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
-export GOPATH=$(go env GOPATH)
-
 SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
-pushd "${SCRIPT_ROOT}"
-BOILERPLATE_HEADER="$( pwd )/hack/boilerplate/boilerplate.go.txt"
-popd
-CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; go list -f '{{.Dir}}' -m k8s.io/code-generator)}
 
-# "core:v2alpha1 workload:v2alpha1 apiresource:v2alpha1 tenancy:v2alpha1 tenancy:v1beta1 apis:v2alpha1 scheduling:v2alpha1 topology:v2alpha1" \
-bash "${CODEGEN_PKG}"/generate-groups.sh "deepcopy,client" \
-  github.com/kubestellar/kubestellar/pkg/client github.com/kubestellar/kubestellar/pkg/apis \
-  "edge:v2alpha1" \
-  --go-header-file "${SCRIPT_ROOT}"/hack/boilerplate/boilerplate.generatego.txt \
-  --output-base "${SCRIPT_ROOT}" \
-  --trim-path-prefix github.com/kubestellar/kubestellar
+source "${CODE_GEN_DIR}/kube_codegen.sh"
+
+rm -rf "${SCRIPT_ROOT}/pkg/generated"
```

Comment:
Why do we need to delete the whole directory every time ?  BTW, If so this is a "create" not "update" function.

Reply:
@ezrasilvera this is needed in case you remove an existing API. 
in such a case you want to also remove the generated code for that API and not to leave garbage.